modules/apps/translation/translation-web/src/main/resources/META-INF/resources/import_translation_results.jsp

```diff
@@ -54,121 +54,172 @@ renderResponse.setTitle(LanguageUtil.get(resourceBundle, "import-translation"));
 
 		<clay:container-fluid
 			cssClass="container-view"
+			size="lg"
 		>
-			<clay:sheet
-				cssClass="translation-import-body-form"
-			>
-				<h3 class="mb-4"><%= importTranslationResultsDisplayContext.getFileName() %></h3>
-
-				<c:if test="<%= importTranslationResultsDisplayContext.getSuccessMessagesCount() > 0 %>">
-					<h4 class="text-success">
-						<span class="mr-2">
-							<clay:icon
-								symbol="check-circle-full"
-							/>
-						</span>
-
-						<%= importTranslationResultsDisplayContext.getSuccessMessageLabel(locale) %>
-					</h4>
+			<div class="translation-import-body-form">
 
-					<ul class="list-group list-group-no-bordered list-group-sm">
+				<%
+				boolean importTranslationResultsErrors = importTranslationResultsDisplayContext.getFailureMessagesCount() > 0;
```

Comment:
In Java, we tend to avoid this kind of boolean expressions. You can either use an if statement to initialize it:
```java
boolean importTranslationResultErrors = false;

if (importTranslationResultsDisplayContext.getFailureMessagesCount() > 0) {
    importTranslationResultErrors = true;
}
```
or inline the expression where used, whatever you prefer.